bin/improver-temp-lapse-rate

```diff
@@ -94,14 +101,22 @@ def main():
     orography_cube = load_cube(args.orography_filepath)
     land_sea_mask_cube = load_cube(args.land_sea_mask_filepath)
```

Comment:
There's no need to load the orography and land_sea_mask cubes when args.return_dalr is true. These should move into the else block (the help-strings ought to be updated to reflect this too).

Reply:
I've modified this so the orography and land-sea mask are optional arguments.